Libraries/RhythmGamesBot.Core/RhythmGamesBot.cs

```diff
@@ -8,7 +8,7 @@ namespace RhythmGamesBot.Core
 {
     public class RhythmGamesBot
     {
-        private static Option<DiscordClient> BotInstance { get; set; }
+        private static DiscordClient? BotInstance { get; set; }
```

Comment:
If the class is of a reference type, it automatically can be null, you don't have to explicitly make it nullable

Reply:
and I'm not sure how it works actually, the question mark probably states that it's expected that it can be null, but in this case it's actually not expected to be null

Reply:
Resolved with recent commit